media/services/.env

```
@@ -1,4 +1,21 @@
+# NordVPN/Gluetun Configuration
+# Get your NordVPN credentials from https://account.nordvpn.com
+NORDVPN_USER=
+NORDVPN_PASSWORD=
+
+# VPN Configuration (optional)
+# VPN_TYPE options: openvpn or wireguard (default: openvpn)
+VPN_TYPE=openvpn
+
+# SERVER_COUNTRIES: Country code or name (e.g., "United States", "United Kingdom", "Canada")
+# Leave empty for automatic server selection
+SERVER_COUNTRIES=United States
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)